README.md

```diff
@@ -569,64 +553,22 @@ See below for more information on the experimental drivers.
 ## Experimental Driver Support
 
 Currently Wallaby provides experimental support for both headless chrome and selenium.
-Both of these drivers are still "experimental" because they don't support the full
-api yet and because the implementation is changing rapidly. But, if you would like
-to use them in your project here's what you'll need to do.
-
-### Headless Chrome
-
-In order to run headless chrome you'll need to have chromedriver 2.30 and chrome 60.
-Previous versions of both of these tools _may_ work, but several features will be
-buggy. If you want to setup chrome in a CI environment then you'll still
-need to install and run xvfb. This is due to a bug in chromedriver 2.30 that inhibits
-chromedriver from handling input text correctly. The bug should be fixed in chromedriver 2.31.
-
-If you would like to disable headless mode in chrome you can pass `headless: false` in your config like so:
-
-```elixir
-config :wallaby,
-  chrome: [
-    headless: false
-  ]
-```
-
-### Custom Chromedriver binary
-
-If `chromedriver` is on your `PATH`, then you can skip this step.
-Otherwise (e.g., on NPM-installed `chromedriver` binaries), you can override the path like so:
-
-```elixir
-config :wallaby, chromedriver: "<path/to/chromedriver>"
-```
+Both of these drivers are still "experimental" because they don't support the full API yet and because the implementation is changing rapidly.
+But, if you would like to use them in your project here's what you'll need to do.
 
+Please refer to the [documentation](https://hexdocs.pm/wallaby/Wallaby.Experimental.Chrome.html#content) for further information about using the Chrome driver.
```

Comment:
Small problem: If this is merged but not yet released the docs won't be there. Not sure what a good way to do this is. We can keep it at first, make sub md files or what not :thinking:

Reply:
I think it is fine.

Some projects will specify at the top of the readme that the readme reflects what is on master, and has a link to the readme at the latest commit.

Reply:
I know, not a fan of that as it also requires having that notice on the README and not elsewhere kinda (to be good) :) I also believe that even with that most people will just use the README on master.

One can't protect from it 100% but for big changes where a bigger part of the documentation becomes unavailable I like to be cautious.

Reply:
I can add a note here that says this is unreleased and then remove the note when I do the next release.

Reply:
could also link to the code on master that has the docs. 

It's not super needed imo but I think it'd be nice :)

Or a bigger not (like you said before) at the top of the README noting that configuration has changed in a major way linking to the released README version. Might be the best of all words, big note on top, but only if there were big changes :man_shrugging: